nncf/quantization/algorithms/fast_bias_correction/onnx_backend.py

```diff
@@ -78,7 +80,19 @@ def get_bias_value(node: NNCFNode, nncf_graph: NNCFGraph, model: onnx.ModelProto
 
     @staticmethod
     def get_activation_port_ids_for_bias_node(node: NNCFNode) -> Tuple[int, int]:
-        return 0, 0
+        layer_attrs = node.layer_attributes
+        inputs = deepcopy(layer_attrs.node_attrs["inputs"])
```

Comment:
Why do you utilize input names instead of metatypes attributes such as `bias_port_id` and `weight_port_ids`? It will free storing 'inputs' for each node in its attribute.

Reply:
@kshpv could you clarify your idea? The code snippet will be good.

Reply:
I see that for some metatypes such as Gemm the activation and weight ports are not determined. So this approach is applicable then. 
However, the new proposed layer attribute is not necessary for this function. Because in `layer_attrs` there is a `weight_attrs` mapping from `port_id` to a weight info. 
So, the information of the weight port (from `weight_attrs`) and bias port (from metatype) is enough for detecting the activation port, am I right?
If this is correct my suggestion is to remove adding new layer_attrs in these changes for ONNX
@alexsu52